docs/ethical-advertising.rst

```diff
@@ -4,14 +4,35 @@ Ethical Advertising
 Read the Docs is a large,
 free web service.
 There is one proven business model to support this kind of site:
-Advertising.
-We're building an advertising model that respects users while providing value to advertisers.
+**Advertising**.
+However,
+instead of using Google,
+we are building the advertising model that we want to exist.
+We only work with companies we like and respect,
+and offer products that are relevent to developers
```

Comment:
Yep, I like this better. 

Though I still disagree with the assertion, "There is one proven business model" -- there are other business models than advertising, and some are even in place, like site membership fees. So maybe address that earlier, here? Or is this page a standalone explanation only for ads?

WTD has available site membership fees, which I hadn't even realized until I scrolled to the bottom of this page, maybe move higher? Also I wonder if the paid site should be featured up higher as well?

How about brokerage (which may be what this model can be called, bringing providers and users together)? Or did you ever consider data mining (if you can't see the product your data is the product)? Or percentage pay by traffic tiers? Considered and rejected because reasons? (such as, scale, perceived value prop, privacy concerns, etc).

Make sure to spell check relevant - and add a period at the end of the sentence.